recipes/smif/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "smif" %}
+{% set version = "0.5.0" %}
+{% set sha256 = "a806ef66b785e1620300bc9ce55a5488fc77d393ea3175db661cc741e234acf0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - smif = smif.cli:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyscaffold
+  run:
+    - fiona 1.7.10
+    - flask 0.12.2
+    - isodate 0.6.0
+    - networkx 1.11
+    - numpy 1.13.1
+    - pint 0.8.1
+    - python
+    - python-dateutil 2.6.1
+    - pyyaml 3.12
+    - rtree 0.8.3
+    - scikit-optimize 0.3
+    - shapely 1.6.2
+
```

Comment:
You need tests for both python imports as well as to test the cli:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-imports
https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

for the imports, please test importing all of the sub-packages.

Reply:
What is a suitable level of testing here? Should the command just be `smif --help`?